src/core/test/shared/cli/cli-with-multiple-names.ts

```diff
@@ -3,6 +3,7 @@ import { CLI, Command } from '@orbital/core';
 // tslint:disable:no-empty
 
 // TODO: These are copy pasted from an other cli file ! USE TEH SHARED ONE
+// for the sake of testing there is no need to be that DRY. You can refactor yourself if you want :)
```

Comment:
I'm really not of this opinion, the tests should be as clean as the rest of the code. What if they are unreadable terribly long and their are pieces everywhere ? We'll have a hard time maintaining them and this is not what we want for any software.

Reply:
that's fair. we can revisit this soon

Reply:
Continuously :smile: